matter_server/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 3.0.0
+
+- Bump Matter Server to 2.0.1
+- Use Nabu Casa Vendor ID by default (requires recommissioning of devices)
```

Comment:
Should we be a bit more explicit?

```suggestion
- Use Nabu Casa Vendor ID by default (requires recommissioning of devices)

### Breaking

- All commissioned devices need to be recommissioned.
```

Reply:
Yeah a bit more explicit seems sensible. I guess then we can drop it from the brackets above.

```suggestion
- Use Nabu Casa Vendor ID by default

### Breaking

- All commissioned devices need to be recommissioned.
```